### DISCUSSION TITLE
Not visible tun0 interface

### DISCUSSION THREAD

**Discussion by erikpinheiro** [strongswan/strongswan#446] [General]
I have a doubt, I created a strongswan service, but I don't have tun0 interface in ifcong command. Tunnel is established with remote, but it has no traffic, no route. What should I do to make it work?

thanks

**Comment by tobiasbrunner** [comment]
Unless you use the [kernel-libipsec](https://wiki.strongswan.org/projects/strongswan/wiki/kernel-libipsec) plugin to use IPsec in userland or you build some other [route-based VPN](https://wiki.strongswan.org/projects/strongswan/wiki/RouteBasedVPN), there are no interfaces involved because the kernel handles IPsec traffic directly via IPsec policies. You might want to read [IntroductionTostrongSwan](https://wiki.strongswan.org/projects/strongswan/wiki/IntroductionTostrongSwan).
